popup.js

```diff
@@ -0,0 +1,126 @@
+    Year: '2015',
+    description:
+      'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    live: 'https://fombi-favour.github.io/Microverse-portfolio/',
+    source: 'https://github.com/Fombi-Favour/Microverse-portfolio',
+  },
+  {
+    id: 1,
+    title: 'Uber Navigation',
+    image: './assets/project2.svg',
+    author: 'Uber',
+    spec: 'Full Stack Dev',
+    Year: '2015',
+    description:
+      'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    technologies: ['HTML', 'Ruby on Rails', 'CSS', 'JavaScript'],
+    live: 'https://fombi-favour.github.io/Microverse-portfolio/',
+    source: 'https://github.com/Fombi-Favour/Microverse-portfolio',
+  },
+  {
+    id: 2,
+    title: 'Facebook 360',
+    image: './assets/project3.svg',
+    author: 'FACEBOOK',
+    spec: 'Full Stack Dev',
+    Year: '2015',
+    description:
+      "Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.",
+    technologies: ['HTML', 'Ruby on Rails', 'CSS', 'JavaScript'],
+    live: 'https://fombi-favour.github.io/Microverse-portfolio/',
+    source: 'https://github.com/Fombi-Favour/Microverse-portfolio',
+  },
+  {
+    id: 3,
+    title: 'Multi-Post Stories',
+    image: './assets/project4.svg',
+    author: 'FACEBOOK',
+    spec: 'Full Stack Dev',
+    Year: '2015',
+    description:
+      'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    technologies: ['HTML', 'Ruby on Rails', 'CSS', 'JavaScript'],
+    live: 'https://fombi-favour.github.io/Microverse-portfolio/',
+    source: 'https://github.com/Fombi-Favour/Microverse-portfolio',
+  },
+];
+
+details.forEach((detail) => {
+  projects.innerHTML += `
+    <main class="project-row" id='${detail.id}'>
+        <div class="left">
+          <img src="${detail.image}" alt="project-1" />
+        </div>
+        <div class="right">
+          <h1>${detail.title}</h1>
+          <div class="content">
+            <h3 class="author">${detail.author}</h3>
+            <img src="./assets/Counter.svg" alt="dot" />
+            <h3>${detail.spec}</h3>
+            <img src="./assets/Counter.svg" alt="dot" />
+            <h3>${detail.Year}</h3>
+          </div>
+          <p>
+            ${detail.description}
+          </p>
+          <ul class="tag">
+            ${detail.technologies.map((tag) => `<li>${tag}</li>`).join('')}
+          </ul>
+          <button type="button" id="${detail.id}" class="btn-see-detail">See project</button>
+        </div>
+      </main>
+    `;
+});
+
+function showDetails(Id) {
+  Title.textContent = details[Id].title;
+  Author.textContent = details[Id].author;
+  spec1.innerHTML = details.spec;
+  spec2.innerHTML = details.Year;
```

Comment:
- [ ] Kindly consider rendering the specs data properly as it currently returning `undefined`. This is because you're not passing the `Id` parameter of the passing so the data is not being reached. This will ensure your data is displayed so we don't get our users wondering what's `undefined` 😂. See below for more details: 

<table>
<tr>
<th align="middle">Figma design</th>
<th align="middle">Your design</th>
</tr>
<tr>
<td>
<img width="337" alt="Screenshot 2023-05-19 at 10 58 44" src="https://github.com/Fombi-Favour/Microverse-portfolio/assets/107192076/e2fc1cd3-8ac7-42a6-a61e-f62b483e0add">
</td>
<td>
<img width="330" alt="Screenshot 2023-05-19 at 10 34 31" src="https://github.com/Fombi-Favour/Microverse-portfolio/assets/107192076/ff1a792b-6c2d-4487-8556-43506b84f6f7">
</td>
</tr>
</table>